### DISCUSSION TITLE
Is there a way to multi delete text on layers when multi selecting layers

### DISCUSSION THREAD

**Discussion by 8-8Sophie8-8** [keiwando/toonsquid#706] [Q&A] [answered]
So I have numbered on some drawings then extended these drawings, have work hours on the animation then after I done I  create a symbol with it. 

Then I realized I forgot to remove the numbers on the drawings before extending them( extending options duplicate the drawings) .

WHAT TO DO ?

I rasterized the symbol it gave me more than 1000 thousand drawing to remove the text on.

then I told myself if I multi select all the drawings, then select the text, then click delete it would delete that text on every drawings but no , it did not work . 

It would take me hours to remove the text 1 by 1 on each drawing , ( I remove it on 180 drawings and it took me 40 min ) I had an alarm set to 1 hour while manually erasing the text unto the drawings  )

the text is on the same spot on every copy of the drawing , but yet I can’t multi erase it when multi select drawings.

is there a way to do this to not have to manually erase everything 1 by 1 ?

**Comment by Cat-Butler** [comment]
Rasterizing it breaks the chance to edit the symbol/drawings directly. If you have done this and you no longer have the original symbol, it might be easier to export the animation as an image sequence and edit in a video editing app.

If you still have the original symbol in your Library, open the Library, tap-and-hold on the symbol in the Library, and you will see a menu appearing across the lower edge of the Library. Tap on Edit in that menu, and you will get the original timeline for the symbol, and you can then edit the original extended drawings.

**Reply by 8-8Sophie8-8** [reply]
When I Export  as image sequence  all extended drawing are also exported which mean I would still have over 1000 drawings to erase text on , and have to upload all these drawings into the art program to be able to erase the text on all of them at once and the art program doesn’t even allow multiple import, only 1 by 1 import and when it’s time to export, the art program doesn’t offer multiple export only 1 by 1 export. 


( I am not sure if I understand correctly the explanation below much ) but when I enter the menu to edit the symbol l cannot delete nor erase anything in there, don’t know why .

**If you still have the original symbol in your Library, open the Library, tap-and-hold on the symbol in the Library, and you will see a menu appearing across the lower edge of the Library. Tap on Edit in that menu, and you will get the original timeline for the symbol, and you can then edit the original extended drawings.**


https://github.com/user-attachments/assets/99db0de8-2f58-4be5-88fe-f3fd0344baea

**Reply by Cat-Butler** [reply]
Some _video editing_ apps have extended multi-frame functionality (can treat multiple frames/images as a single-file movie). I would not recommend doing it in an art (single image) program.

I suspect that you would have to show the layers panel to see why the eraser is not erasing. For example: Are there multiple layers? Is the layer locked?

**Comment by 8-8Sophie8-8** [comment]
Eraser not working when it comes to clip ( symbol) … when it is a clip I somehow cannot erase nor delete anything on the clip in edit mode. But if I paint or add things on that clip in the edit mode, I am able to erase and delete things I added but not things that were already there.

this seems to happen only with clip but if it is a drawing and not a symbol I can erase anythings in it  when in edit mode .

https://github.com/user-attachments/assets/7e8a5752-5670-4ef9-8ea6-80a38b794659

**Comment by Cat-Butler** [comment]
Which brush are you using for the eraser?

Please check/record with the layer panel showing. Are you on the correct layer? Is the layer locked?
![layers](https://github.com/user-attachments/assets/642c8cd6-2461-4f60-8af1-356bfe0bb00e)

**Comment by 8-8Sophie8-8** [comment, marked as answer]
The option to erase stuff on clip have been found …thanks 
So what I did is: 1) I select the symbol clip inside the library .

2) then click on the edit button  then I click on the layer to open the option menu , then click on (edit clip ) then I enter a screen where the clip can directly be edited . 

https://github.com/user-attachments/assets/95f294f0-f566-4181-86c2-89b472c720cc
